.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Code for Hawaii
+    url: https://github.com/CodeforHawai/discussions
+    about: Start a conversation here
+  - name: Code for Hawaii website
+    url: https://codeforhawaii.org
+    about: Contact us through our website form
```

Comment:
We have a form? 😃

Reply:
We do not lol it got stripped out because it didn't work

Reply:
Forgot about that. My b. 
Should we/I get one working or nah?